packages/d3/d3.0.3.0/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+maintainer: "Spiros Eliopoulos <seliopou@gmail.com>"
+authors: "Spiros Eliopoulos <seliopou@gmail.com>"
+homepage: "https://github.com/seliopou/ocaml-d3"
+bug-reports: "https://github.com/seliopou/ocaml-d3/issues"
+license: "BSD-3-clause"
+dev-repo: "https://github.com/seliopou/ocaml-d3.git"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+build-test: [
+  ["ocaml" "setup.ml" "-configure" "--enable-tests"]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-test"]
+]
+build-doc: ["ocaml" "setup.ml" "-doc"]
+remove: ["ocamlfind" "remove" "d3"]
+depends: [
+  "js_of_ocaml" {>= "3.0.0"}
```

Comment:
due to a the upstream versioning, this actually needs to be `>= "3.0"`.  The extra .0 causes it to fail the intended constraint.